x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicators_table/indicators_table.stories.tsx

```diff
@@ -35,20 +37,22 @@ export function WithIndicators() {
 
   return (
     <KibanaReactContext.Provider>
-      <IndicatorsTable
-        browserFields={{}}
-        loading={false}
-        pagination={{
-          pageSize: 10,
-          pageIndex: 0,
-          pageSizeOptions: [10, 25, 50],
-        }}
-        indicators={indicatorsFixture}
-        onChangePage={stub}
-        onChangeItemsPerPage={stub}
-        indicatorCount={indicatorsFixture.length * 2}
-        indexPattern={mockIndexPattern}
-      />
+      <IndicatorsFiltersContext.Provider value={mockIndicatorsFiltersContext}>
```

Comment:
this `IndicatorsFiltersContext` component allows us to share `filterManager` (received from the `KQLBar`) between all the components inside the `IndicatorsTable` (like `CellActions` and `IndicatorsFlyoutTable`)